packages/core/src/icon/icon.element.scss

```diff
@@ -36,126 +36,6 @@ svg {
   display: block;
 }
 
-// sizing
-// DEPRECATED IN 4.0: all legacy sizing classnames
-// will be replaced with [size="sm|md|lg|xl|xxl"] selectors!
-:host(.clr-i-size-sm) {
-  // weirdly, the default... 16px
-  @include equilateral(#{$cds-token-space-size-7});
-}
-
-:host(.clr-i-size-md) {
-  // 24px
-  @include equilateral(#{$cds-token-space-size-9});
-}
-
-:host(.clr-i-size-lg) {
-  // 36px
-  @include equilateral(#{$cds-token-space-size-11});
-}
-
-:host(.clr-i-size-xl) {
-  // 48px
-  @include equilateral(#{$cds-token-space-size-12});
-}
-
-:host(.clr-i-size-xxl) {
-  // 64px
-  @include equilateral(calc(#{$cds-token-space-size-13} - #{$cds-token-space-size-5}));
-}
-
-// colors
-// DEPRECATED IN 3.0: is-green, .is-success
-:host(.is-green),
-:host(.is-success),
-:host([status='success']) {
```

Comment:
@mathisscott When I went through the changes I think lines like 71, 79, 85, etc actually don't need to be removed. Am I wrong?

Reply:
You are correct. Good catch. we do still need the `:host([status='...'])` styles...